.github/workflows/daily-work.yaml

```diff
@@ -0,0 +1,47 @@
+name: Daily work
+
+on:
+  workflow_run:
+    workflows: ['e2e tests']
+    types: [completed]
+
+jobs:
+  build-operator:
+    runs-on: ubuntu-latest 
+    if: ${{ github.event.workflow_run.conclusion == 'success' && contains('server daily version', github.event.workflow_run.display_title) }}  
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
This doesn't need to be in a new workflow. Just add one step in e2e-tests, then get the operator version from "build.outputs.operator-image". We don't need to download the repo and infer the versions again.

Reply:
I thought we want to build a new daily work action to control the upload of the image. It is actually much easier to just add one new job in the e2e.yml. If that is what we want I will do that, please let me know

Reply:
Moved the task to e2e.yml in https://github.com/vertica/vertica-kubernetes/pull/929/commits/61989734fa878a5b2719d6dcf2f048ae4a10a3be